packages/devtools_app/lib/src/shared/status_line.dart

```diff
@@ -67,22 +67,21 @@ class StatusLine extends StatelessWidget {
   }
 
   List<Widget> _getStatusItems(BuildContext context, bool showIsolateSelector) {
-    final isExtraNarrow = ScreenSize(context).width == MediaSize.xxs;
-    final isNarrow = isExtraNarrow || ScreenSize(context).width == MediaSize.xs;
+    final screenWidth = ScreenSize(context).width;
     final Widget? pageStatus = currentScreen.buildStatus(context);
     return [
-      buildHelpUrlStatus(context, currentScreen, isNarrow),
+      buildHelpUrlStatus(context, currentScreen, screenWidth),
       const BulletSpacer(),
-      if (!isExtraNarrow && showIsolateSelector) ...[
+      if (screenWidth > MediaSize.xxs && showIsolateSelector) ...[
         const IsolateSelector(),
         const BulletSpacer(),
       ],
-      if (!isNarrow && pageStatus != null) ...[
+      if (screenWidth > MediaSize.xs && pageStatus != null) ...[
         pageStatus,
         const BulletSpacer(),
       ],
-      buildConnectionStatus(context, isExtraNarrow),
-      if (isEmbedded && !isExtraNarrow) ...[
+      buildConnectionStatus(context, screenWidth),
+      if (screenWidth > MediaSize.xxs && isEmbedded) ...[
```

Comment:
nit: we could put `screenWidth > MediaSize.xxs` since it is used twice

Reply:
Good point, done!